_events/2021/2021-02-career-panel.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "ADSA/US-RSE Early Career Panel"
+expires: 2021-02-23
```

Comment:
Should we expire one day after the event (in case someone is looking for it the day of?)

Reply:
Done